README.md

```diff
@@ -609,6 +614,170 @@ Returns results in the format...
 
 NOTE: All predicates with the URI as the subject will be included under "predicates" key.  The selected keys are determined by the configuration file and can be one or more of id_predicate, label_predicate (required), altlabel_predicate, sameas_predicate, narrower_predicate, or broader_predicate.
 
+#### Linked Data Authority Usage Notes
+##### Agrovoc Linked Data
+
+Configuration File: [agrovoc.json](https://github.com/projecthydra/questioning_authority/blob/master/config/authorities/linked_data/agrovoc.json)
+
+*Example Search URLs*
+
+Pattern:  `http://www.your.host/qa/search/linked_data/agrovoc?q={?query}&lang={?language_code}`
+
+Localhost Examples:
+<br>http://localhost:3000/qa/search/linked_data/agrovoc?q=hive
+<br>http://localhost:3000/qa/search/linked_data/agrovoc?q=ruche&lang=fr
+
+where,
+
+| parameter | required? | description | default value | example values |
+| --------- | --------- | ----------- | ------------- | -------------- |
+| {?query}  | required  | the query string to send to the authority | N/A | cornell, twain, ezra, etc. |
+| {?language_code} | optional | results will be filtered to the language code | en | en, fr, de, etc. |
+
+*Example Fetch Term URLs*
+
+Pattern:  `http://www.your.host/qa/show/linked_data/agrovoc/{?term_id}`
+
+Localhost Examples:
+<br>http://localhost:3000/qa/show/linked_data/agrovoc/c_3655
+
+where,
+
+| parameter  | required? | description | default value | example values |
+| ---------  | --------- | ----------- | ------------- | -------------- |
+| {?term_id} | required  | the id of the term to fetch from the authority | N/A | c_123 |
+
+NOTE: A default language of `['en']` is set in the configuration for term fetch.  You can change the default language by copying the entire configuration file from QA /config/authorities/linked_data/agrovoc.json to your app in the same location.  The language can be set to one or more languages by using the standard language codes in an array, e.g. `['fr']` or `['en', 'fr']`, etc.
+
+
+##### Geonames Linked Data
+
+Configuration File: [geonames.json](https://github.com/projecthydra/questioning_authority/blob/master/config/authorities/linked_data/geonames.json)
+
+*Example Search URLs*
+
+Pattern:  `http://www.your.host/qa/search/linked_data/geonames?q={?query}&maximumRecords={?maximumRecords}&username={?username}`
+
+Localhost Examples:
+<br>http://localhost:3000/qa/search/linked_data/geonames?q=ithaca&username=yourusername
```

Comment:
You can set the username's default value, which I should probably mark as recommended.  Since we dumped the merging code, this means that the entire config file has to be copied to the app and changes made to the copy.

There is a comment in the README at line 673 that states how to do this.
https://github.com/samvera/questioning_authority/blob/geonames/README.md#geonames-linked-data
Look at the note after the param description table.